hack/Containerfile

```diff
@@ -0,0 +1,9 @@
+# This file is mostly s2i-python with a static assemble script (pip install)
+FROM PLACEHOLDER_NOT_AN_IMAGE
+# This will systematically be replaced by buidah `--from` option
+
+USER root
```

Comment:
That's the containerfile produced by s2i when using --as-dockerfile (with minor adjustements)
I think it's for the chown line below -> files need to be group-owned by the root group, in order to support runtime-assigned uid for the container (which openshift does by default if I remember correctly).

But since the running container will never need to access that `requirements.txt`, we can probably get rid of that, indeed